<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0" stdsetdef="1">
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>KEnterScheduleDlg</class>
  <widget class="QDialog" name="KEnterScheduleDlg">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>679</width>
        <height>410</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>Enter Schedule</string>
    </property>
    <property name="sizeGripEnabled">
      <bool>true</bool>
    </property>
    <layout class="QVBoxLayout">
      <item>
        <widget class="QGroupBox" name="groupBox1">
          <property name="title">
            <string>Schedule Details</string>
          </property>
          <layout class="QHBoxLayout">
            <item>
              <widget class="QLabel" name="textLabel5">
                <property name="sizePolicy">
                  <sizepolicy>
                    <hsizetype>5</hsizetype>
                    <vsizetype>5</vsizetype>
                    <horstretch>1</horstretch>
                    <verstretch>1</verstretch>
                  </sizepolicy>
                </property>
                <property name="minimumSize">
                  <size>
                    <width>70</width>
                    <height>0</height>
                  </size>
                </property>
                <property name="font">
                  <font/>
                </property>
                <property name="text">
                  <string comment="@label schedule name">Name:</string>
                </property>
                <property name="wordWrap">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QLabel" name="m_scheduleName">
                <property name="text">
                  <string/>
                </property>
                <property name="wordWrap">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item>
            <spacer name="Horizontal Spacing3">
              <property name="sizeHint">
                <size>
                  <width>20</width>
                  <height>0</height>
                </size>
              </property>
              <property name="sizeType">
                <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="orientation">
                <enum>Qt::Horizontal</enum>
              </property>
            </spacer>
            </item>
            <item>
              <widget class="QLabel" name="textLabel1_2">
                <property name="sizePolicy">
                  <sizepolicy>
                    <hsizetype>5</hsizetype>
                    <vsizetype>5</vsizetype>
                    <horstretch>1</horstretch>
                    <verstretch>1</verstretch>
                  </sizepolicy>
                </property>
                <property name="minimumSize">
                  <size>
                    <width>70</width>
                    <height>0</height>
                  </size>
                </property>
                <property name="font">
                  <font/>
                </property>
                <property name="text">
                  <string>Type:</string>
                </property>
                <property name="wordWrap">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QLabel" name="m_type">
                <property name="sizePolicy">
                  <sizepolicy>
                    <hsizetype>5</hsizetype>
                    <vsizetype>5</vsizetype>
                    <horstretch>1</horstretch>
                    <verstretch>1</verstretch>
                  </sizepolicy>
                </property>
                <property name="text">
                  <string/>
                </property>
                <property name="wordWrap">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
      <item>
        <widget class="KMyMoneyRegister::Register" name="m_register">
          <property name="rowCount">
            <number>0</number>
          </property>
          <property name="columnCount">
            <number>12</number>
          </property>
          <column>
            <property name="text">
              <string>Security</string>
            </property>
          </column>
          <column>
            <property name="text">
              <string>Details</string>
            </property>
          </column>
          <column>
            <property name="text">
              <string>C</string>
            </property>
          </column>
          <column>
            <property name="text">
              <string>Payment</string>
            </property>
          </column>
          <column>
            <property name="text">
              <string>Deposit</string>
            </property>
          </column>
          <column>
            <property name="text">
              <string>Quantity</string>
            </property>
          </column>
          <column>
            <property name="text">
              <string>Price</string>
            </property>
          </column>
          <column>
            <property name="text">
              <string>Value</string>
            </property>
          </column>
          <column>
            <property name="text">
              <string>Balance</string>
            </property>
          </column>
        </widget>
      </item>
      <item>
        <widget class="QGroupBox" name="frame4">
          <layout class="QVBoxLayout">
            <property name="margin">
              <number>1</number>
            </property>
            <item>
              <widget class="KMyMoneyTransactionForm::TransactionForm" name="m_form">
                <property name="sizePolicy">
                  <sizepolicy>
                    <hsizetype>5</hsizetype>
                    <vsizetype>5</vsizetype>
                    <horstretch>1</horstretch>
                    <verstretch>5</verstretch>
                  </sizepolicy>
                </property>
                <property name="rowCount">
                  <number>5</number>
                </property>
                <property name="columnCount">
                  <number>1</number>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
      <item>
        <widget class="Line" name="line2">
          <property name="frameShape">
            <enum>QFrame::HLine</enum>
          </property>
          <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
          </property>
        </widget>
      </item>
      <item>
        <layout class="QHBoxLayout">
          <item>
            <widget class="QPushButton" name="buttonHelp">
              <property name="text">
                <string>Help</string>
              </property>
            </widget>
          </item>
          <item>
            <spacer name="Horizontal Spacing2">
              <property name="sizeHint">
                <size>
                  <width>190</width>
                  <height>20</height>
                </size>
              </property>
              <property name="sizeType">
                <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="orientation">
                <enum>Qt::Horizontal</enum>
              </property>
            </spacer>
          </item>
          <item>
            <widget class="QPushButton" name="buttonOk">
              <property name="text">
                <string>Enter</string>
              </property>
              <property name="shortcut">
                <string/>
              </property>
              <property name="autoDefault">
                <bool>true</bool>
              </property>
              <property name="default">
                <bool>true</bool>
              </property>
              <property name="toolTip" stdset="0">
                <string>Enter the transaction and advance the next due date of this schedule to the next payment date.</string>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QPushButton" name="buttonSkip">
              <property name="text">
                <string>&amp;Skip</string>
              </property>
              <property name="isDragEnabled" stdset="0">
                <bool>true</bool>
              </property>
              <property name="toolTip" stdset="0">
                <string>Do not enter the transaction but advance the next due date of this schedule to the next payment date.</string>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QPushButton" name="buttonIgnore">
              <property name="text">
                <string>Ignore</string>
              </property>
              <property name="toolTip" stdset="0">
                <string>Do not enter or skip payments for this schedule but continue with the next schedule.</string>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QPushButton" name="buttonCancel">
              <property name="text">
                <string>Cancel</string>
              </property>
              <property name="toolTip" stdset="0">
                <string>Cancel processing of schedule entry.</string>
              </property>
            </widget>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11"/>
    <connections>
    <connection>
      <sender>buttonOk</sender>
      <signal>clicked()</signal>
      <receiver>KEnterScheduleDlg</receiver>
      <slot>accept()</slot>
    </connection>
    <connection>
      <sender>buttonCancel</sender>
      <signal>clicked()</signal>
      <receiver>KEnterScheduleDlg</receiver>
      <slot>reject()</slot>
    </connection>
  </connections>
</ui>
